### DISCUSSION TITLE
Public Key

### DISCUSSION THREAD

**Discussion by thawkes70** [thawkes70/Tafe22S219#5] [General]
Could you please send me your public key from Puttygen.  Once I save that to GitHub you should have access

**Comment by Chitusnair** [comment]
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCcTbsSsTJKUWpXUJ90obmA/p4ZcaEYuvxswdTST4qAFN+5cljId7BT+q0q38ybYpW9fUTQAs4sWUauJANYv1gSApq5ENrivMgOt5uqWIo0yOXBo6L7Eosy8zleMBfHM42WtAWS7PEPd67xnbFcFvQoKq+3qXs+zNwSjgKH8xB+M72ZXk0r7fxSP5P3TdEsMggmPMhaL3Bqby94SdX8dW4LXzsJHd3Mtmqjt1A1qw+vBXn9V22N80PNswuU0FE54OVPSqSVJGQ+lM4nhipfxaPsEB/OaoVkqMefjhWSf1oOV/f/JjZ0SsXZ2Hw8XzQ5cNrlpVclX+rar6XZ04Ifdgzf rsa-key-20220901

**Comment by Chitusnair** [comment]
I have shared the public key. I am not able to see the Repo URL now. Once I get the access I hope I will see the URL and then I will add the Remote settings on Tortoise and then do a Git sync to the newly created folder for the project. Am I right?

**Comment by thawkes70** [comment]
I have added your key, hopefully you should have access now.  Not sure why you couldn't see the Repo, let me know how you go now I have added the key.  Yes once you have access to the branches you should be able to to a Git Sync to update your local repository.

**Reply by Chitusnair** [reply]
Thanks, I have cloned the repository to my local folder.
![Capture](https://user-images.githubusercontent.com/29492335/187830635-3904509f-4efe-4c52-af36-e32696a008d5.JPG)
